build.proj

```diff
@@ -35,7 +39,7 @@
           Outputs="@(RestoreDotnetCliBuildFrameworkOutputs)">
 
     <PropertyGroup>
-      <ExtraRestoreArgs Condition=" '$(GeneratingPropsFile)' == 'true' ">$(ExtraRestoreArgs) /p:GeneratingPropsFile=$(GeneratingPropsFile)</ExtraRestoreArgs>
+      <ExtraRestoreArgs Condition=" '$(GeneratePropsFile)' == 'true' ">$(ExtraRestoreArgs) /p:GeneratePropsFile=$(GeneratePropsFile)</ExtraRestoreArgs>
```

Comment:
I think the Condition can be dropped here, and just always put `/p:GeneratePropertyFile=$(GeneratePropsFile)` on the args.